users/stanrc85/stanrc85.c

```diff
@@ -14,23 +14,21 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
       clear_mods();
       clear_oneshot_mods();
       send_string_with_delay_P(PSTR("make " QMK_KEYBOARD ":" QMK_KEYMAP), 10);
-        if (temp_mod & MODS_SHIFT_MASK || temp_osm & MODS_SHIFT_MASK ) {
+        if (temp_mod & MOD_MASK_SHIFT || temp_osm & MOD_MASK_SHIFT ) {
           //RESET board for flashing if SHIFT held or tapped with KC_MAKE
           #if defined(__arm__)
             send_string_with_delay_P(PSTR(":dfu-util"), 10);
-            wait_ms(100);
-            reset_keyboard();
           #elif defined(BOOTLOADER_DFU)
             send_string_with_delay_P(PSTR(":dfu"), 10);
           #elif defined(BOOTLOADER_HALFKAY)
             send_string_with_delay_P(PSTR(":teensy"), 10);
           #elif defined(BOOTLOADER_CATERINA)
             send_string_with_delay_P(PSTR(":avrdude"), 10);
-          #else
-            reset_keyboard();
           #endif // bootloader options
+          send_string_with_delay_P(PSTR(SS_TAP(X_ENTER)), 10);
+          reset_keyboard();
         }
-        if (temp_mod & MODS_CTRL_MASK || temp_osm & MODS_CTRL_MASK) { send_string_with_delay_P(PSTR(" -j8 --output-sync"), 10); }
+        if (temp_mod & MOD_MASK_CTRL || temp_osm & MOD_MASK_CTRL) { send_string_with_delay_P(PSTR(" -j8 --output-sync"), 10); }
```

Comment:
Same reasoning as above.
```suggestion
        if (mods & MOD_MASK_CTRL) {
          send_string_with_delay_P(PSTR(" -j8 --output-sync"), 10);
        }
```
I also suggest adding some line breaks to make it a bit easier on the eyes 🙂

Reply:
Done!